### PR TITLE
Sdc gallery app, multiple choice (check boxes) component workflow

### LINKED ISSUE #1011: Catalog app, multiple choice (check boxes) component workflow
**Is your feature request related to a problem? Please describe.**
1. User click on multiple choice component.
2. It launches multiple choice screen with questionnaire, which demonstrate multiple choice uses.
3. click on navigation back key in action bar, it takes user back on landing page.

Design multiple choice questionnaire json.
Write api to fetch multi choice questionnaire json file name.
Apply custom style at application side.

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Would you like to work on the issue?**
Please state if this issue should be assigned to you or who you think could help to solve this issue.

### PR DESCRIPTION
**Pull request by santosh-pingle**
**IMPORTANT: All PRs must be linked to an issue (except for extremely trivial and straightforward changes).**

Fixes #1011 

**Description**
Design multiple choice questionnaire json.
Write api to fetch multi choice questionnaire json file name.
Apply custom style at application side.

**Alternative(s) considered**
Have you considered any alternatives? And if so, why have you chosen the approach in this PR?

**Type**
Choose one: Feature

**Screenshots (if applicable)**

**Checklist**
- [x] I have read and acknowledged the [Code of conduct](https://github.com/google/android-fhir/blob/master/docs/CODE_OF_CONDUCT.md)
- [x] I have read [How to Contribute](https://github.com/google/android-fhir/blob/master/docs/contributing.md)
- [x] I have read the [Developer's guide](https://github.com/google/android-fhir/wiki/Developer's-Guide)
- [x] I have signed the Google [Individual CLA](https://cla.developers.google.com/about/google-individual), or I am covered by my company's [Corporate CLA](https://cla.developers.google.com/about/google-corporate )
- [x] I have discussed my proposed solution with code owners in the linked issue(s) and we have agreed upon the general approach
- [x] I have run `./gradlew spotlessApply` and `./gradlew spotlessCheck` to check my code follows the style guide of this project
- [x] I have run `./gradlew check` and `./gradlew connectedCheck` to test my changes locally
- [x] I have built and run the reference app(s) to verify my change fixes the issue and/or does not break the reference app(s)

### CODE REVIEW

**Comment by santosh-pingle** [comment]
Screen Shot 2022-01-13 at 9 29 06 PM

File: catalog/src/main/res/values/styles.xml [hunk 1]
```diff
@@ -73,6 +77,20 @@
...
+        <item name="android:textColor">@color/checked_text_color</item>
+        <item
+            name="materialThemeOverlay"
+        >@style/ThemeOverlay.App.CheckBox</item>
+    </style>
+
+    <style name="ThemeOverlay.App.CheckBox" parent="">
+        <item name="colorSecondary">?attr/colorPrimary</item>
+    </style>
+
```
Comment (jingtang10) [inline catalog/src/main/res/values/styles.xml:93]: can we make this the default in the data capture lib?

Comment (jingtang10) [reply catalog/src/main/res/values/styles.xml:93]: same thing with the radio button.

sorry i didn't catch this in the previous pr.

i don't see any reason we shouldn't make this the default?

**Review by jingtang10 — CHANGES_REQUESTED** [review]

File: datacapture/src/main/res/layout/questionnaire_item_boolean_type_picker_view.xml [hunk 2]
```diff
@@ -59,19 +59,15 @@
             android:id="@+id/yes_radio_button"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:textColor="?attr/questionnaireTextColor"
```
Comment (jingtang10) [inline datacapture/src/main/res/layout/questionnaire_item_boolean_type_picker_view.xml:62]: can you also remove this from the `attr.xml` file?

**Review by jingtang10 — APPROVED** [review]
Thanks @santosh-pingle!

**Pull request merged**